### PR TITLE
Update download client default categories

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
NO

#### Description
Updates download client default categories from tv to music

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR
*

### CODE REVIEW

File: src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs [hunk 1]
```diff
@@ -26,7 +26,9 @@ public NzbgetSettings()
         {
             Host = "localhost";
             Port = 6789;
-            TvCategory = "tv";
+            TvCategory = "Music";
+            Username = "nzbget";
+            Password = "tegbzn6789";
```
Comment (runraid) [inline src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs]: What's this password?  (generally password should never be in source)

Comment (Qstick) [inline src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs]: Thats the default password for nzbget. I'm good with getting rid of it, just saves most users a step.

**Comment by majora2007** [comment]
If it saves the users a step, I vote to keep it in. We can remove later,
but for now, we should stick to what users are used to from Sonarr/Radarr.

**Comment by Qstick** [comment]
I'll resubmit pr this weekend

**Pull request closed without merging**